CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Add `Medium2D` to full simulation in tests.
+- Removed `StableFitter`
+- Stable dispersion fitter is now executed through `plugins.dispersion.web.run`
```

Comment:
maybe we should clarify these bullets a bit, suggestion:
```
- `Fitter` and `StableFitter` unified in a single `Fitter` interface.
- `StableFitter` deprecated, with stable fitter now being run instead through `plugins.dispersion.web.run(Fitter)`.
```
think it makes it a bit more clear how someone should use the stable fitter with the new changes.

Reply:
sorry, I guess one other thing: I noticed the "- Removed `StableFitter`" is still in the changelog, probably this is a little misleading since it's still there, but we already mention that it's deprecated so maybe we can just remove this line?